kustomize/jupyter-web-app/base/config-map.yaml

```diff
@@ -62,7 +62,7 @@ data:
             value: 'workspace-{notebook-name}'
           size:
             # The Size of the Workspace Volume (in Gi)
-            value: '10Gi'
+            value: '32Gi'
```

Comment:
```suggestion
            value: '16Gi'
```

Reply:
If users start installing Tensorflow or whatever into their notebook volume, I'm worried that we'll burn through 16gb quick :/ Are we dead-set on 16gb? I suppose we could always change later.